src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterValue } from '../../types/FilterValue';
+
+interface Props {
+  todos: Todo[],
+  setFilterValue: (value: FilterValue) => void,
+  filterValue: string,
+  deleteTodo: (todoId: number) => void,
+}
+
+export const Footer: FC<Props> = (props) => {
+  const {
+    todos,
+    setFilterValue,
+    filterValue,
+    deleteTodo,
+  } = props;
+
+  const remainingTodosLength = todos.filter(todo => !todo.completed).length;
+  const completedTodosLength = todos.filter(todo => todo.completed).length;
```

Comment:
i dont think u have to filter two times. just do it once and then subtract it from todos.length